### PR TITLE
Remove need to manually index layouts

### PR DESCRIPTION
**Pull request by BlueDrink9**

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt [hunk 1]
```diff
@@ -99,103 +99,108 @@ import com.dessalines.thumbkey.keyboards.KB_UK_RU_MESSAGEEASE_SYMBOLS
...
+    BGPhoneticMessagEaseSymbols(KB_BG_PHONETIC_MESSAGEASE_SYMBOLS),
+    UKMessageEaseSymbols(KB_UK_MESSAGEEASE_SYMBOLS),
+    ENFRMessagEaseSymbols(KB_EN_FR_MESSAGEASE_SYMBOLS),
+    HEMessageEaseSymbols(KB_HE_MESSAGEEASE_SYMBOLS),
+    ENNOMessagEaseSymbols(KB_EN_NO_MESSAGEASE_SYMBOLS),
+
+    ;
+
+    val index: Int
+        get() = ordinal
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt:205]: Good call! But you can just remove this pointless function, and do `.ordinal` where we used to use `index`

**Review by dessalines — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt (hunk 1 above)
Comment (BlueDrink9) [reply app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt]: done

**Review by dessalines — APPROVED** [review]
Nice, thx! Just one conflict from the new keyboard I added. lmk if you'd rather have me handle.

**Comment by BlueDrink9** [comment]
yes please, won't be at computer for a bit

**Review by dessalines — APPROVED** [review]

**Comment by Zwyx** [comment]
Just wanted to let you guys know, I've noticed that this PR replaces some `MessagEase` to `MessageEase`. But `MessagEase` is actually the [correct spelling](https://en.wikipedia.org/wiki/MessagEase) :wink:

**Comment by BlueDrink9** [comment]
Correct, that's a regression on my part. I missed t_e change when fixing conflicts

**Comment by dessalines** [comment]
Open up an issue or PR for that if you would.

**Pull request merged**